### ISSUE TITLE
Empty objects in schema fail the doc generation

### ISSUE THREAD

**Issue by ovandriyanov** [hashicorp/terraform-plugin-docs#100]
Terraform allows providers to have empty objects in their schema, but currently terraform-plugin-docs does not recognize such empty fields neither as required, nor optional, nor read-only, and thus cannot generate docs for such providers and raises an error like the following:

    no match for empty_block_field, this can happen if you have incompatible schema
    defined, for example an optional block where all the child attributes are
    computed, in which case the block itself should also be marked computed

I tried to fix it with this patch: https://github.com/hashicorp/terraform-plugin-docs/pull/99

**Comment by agouil** [comment]
I have the same problem with generating the docs for my provider. I have a field defined like the one below as `Optional` with `MaxItems: 1` and `tfplugindocs` returns the same error message when I run it.

```hcl
resource "provider_resource" "test" {
  name = "test"
  actions {
    some_action {}
  }
}
```
I have defined `some_action` as an empty object because at the same time my provider supports, 

```hcl
resource "provider_resource" "test" {
  name = "test"
  actions {
    another_action {
      field = "foobar"
    }
  }
}
```

**Comment by detro** [comment]
Fixed by #134
